projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/container/multi-step-checkout.component.spec.ts

```diff
@@ -108,6 +108,9 @@ describe('MultiStepCheckoutComponent', () => {
     };
     mockCartService = {
       activeCart$: new BehaviorSubject({}),
```

Comment:
This doesn't represent the actual class correctly - we don't have public properties anymore.